### PR TITLE
Added args to plural()

### PR DESCRIPTION
**Pull request by kyle-seongwoo-jun**
I added `List<String> args` parameter to `plural()`method to use like this Java method. 
[public String getQuantityString (int id, int quantity, Object... formatArgs)](https://developer.android.com/reference/android/content/res/Resources#getQuantityString(int,%20int,%20java.lang.Object...))

### Usage
```
{
  "money": {
    "zero": "{} has no money",
    "one": "{} has {} dollar",
    "many": "{} has {} dollars",
    "other": "{} has {} dollars"
  }
}
```
```
final format = NumberFormat.compact(locale: context.locale.toString());
Text('money').plural(1000000, args: ['John', format.format(1000000)]); // output: John has 1M dollars
```

### CODE REVIEW

**Comment by aissat** [comment]
hi @kyle-seongwoo-jun, great  work need to write a test :+1:

**Comment by kyle-seongwoo-jun** [comment]
@aissat  Hi, i just added test code. Thank you.

**Comment by aissat** [comment]
> @aissat Hi, i just added test code. Thank you.

@kyle-seongwoo-jun  thnx bro, just another thing pleas update readme file added ur doc  :)

**Comment by kyle-seongwoo-jun** [comment]
@aissat I just updated README.md. Thank you.

**Comment by aissat** [comment]
@kyle-seongwoo-jun  nice ;)

**Review by aissat — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -270,7 +277,7 @@ Text('money').plural(1000000, format: NumberFormat.compact(locale: context.local
 print('day'.plural(21)); // output: 21 день
 
 //Static function
-var money = plural('money', 10.23) // output: You have 10.23 dollars
```
Comment (Overman775) [inline README.md:273]: don't need delete this, just stay two variants 
```
//Static function
var money = plural('money', 10.23) // output: You have 10.23 dollars

//Static function with arguments
var money = plural('money_args', 10.23, args: ['John', '10.23'])  // output: John has 10.23 dollars
```

**Review by Overman775 — CHANGES_REQUESTED** [review]

**Comment by Overman775** [comment]
@kyle-seongwoo-jun nice PR 👍

File: README.md (hunk 1 above)
Comment (kyle-seongwoo-jun) [reply README.md:273]: @Overman775 got it. I just update it. Please check it out. Thanks :)

**Review by Overman775 — APPROVED** [review]

**Pull request merged**